near-campaign/src/get_campaign_metadata.rs

```diff
@@ -12,9 +12,13 @@ pub struct Metadata {
 
 #[near_bindgen]
 impl Campaign {
-  pub fn get_campaign_metadata(&self) -> Metadata {
+  pub fn get_campaign_metadata(self) -> Metadata {
```

Comment:
Will it work? Why do you replace &self > self?

Reply:
To avoid having to clone the data within the function. There is no difference in what this function does, just moves `self` into the function to be used (if you look at the generated code, this is the only difference).